starter-code/movies/app.js

```diff
@@ -0,0 +1,52 @@
+var express = require("express");
+var path = require("path");
+var favicon = require("serve-favicon");
+var logger = require("morgan");
+var cookieParser = require("cookie-parser");
+var bodyParser = require("body-parser");
+
+var index = require("./routes/index");
+var users = require("./routes/users");
+const mongoose = require("mongoose");
+
+mongoose.connect("mongodb://localhost/mongoose-movies");
+
+const celebrityRoutes = require("./routes/celebrities");
+
+var app = express();
+
+// view engine setup
+app.set("views", path.join(__dirname, "views"));
+app.set("view engine", "ejs");
+app.use("/celebrities", celebrityRoutes);
```

Comment:
When I'm trying to create a celebrity, I get the error `Cannot read property 'name' of undefined`.
This is probably because you are telling express to use your celebrities router before you are telling it to use`bodyParser`. What is happening is that `req.body` is undefined because the bodyParser is not creating it out of the JSON body.
Make sure you use your routers after the relevant middlewares.